docs/bis/ec/ec-format.md

```diff
@@ -86,6 +87,23 @@ Supported scientific type:
 - `normalized`
 - `zeroNormalized`
 
+**RatioType** determines the type of ratio format to use. Only valid and its required when the type is ratio.
```

Comment:
```suggestion
**RatioType** determines the type of ratio format to use. Only valid and is required when the type is ratio.
```

Reply:
It seems like other attributes of Format uses camelCase, not PascalCase - should ratioType follow the same manner?